requirements.txt

```diff
@@ -12,12 +8,10 @@ django-debug-toolbar==1.9.1
 elasticsearch==5.5.3
 elasticsearch-dsl==5.4.0
 # https://github.com/benoitc/gunicorn/issues/1194#issuecomment-269836343
-gevent==1.3.7
 greenlet==0.4.15
 requests-aws4auth==0.9
 psycopg2-binary==2.7.5
 wagtail>=1.13.1,<1.14
 django-allauth==0.37.1
 
-git+https://github.com/OpenUpSA/wazimap.git@releases#egg=wazimap
```

Comment:
```
wagtail>=1.13.1,<1.14
django-allauth==0.37.1
```
Needs to go, `hurumap-dashboard` installs them. I'm not sure about `greenlet`, I think `gunicorn[geven]` installs `greenlet` but double check.